docs/data/sources/segment.md

```diff
@@ -116,18 +126,18 @@ For examples of how your code should look, see the [Android demo code](https://g
 
 Follow the appropriate instructions in Segment's [documentation](https://segment.com/docs/sources/).
 
-## Mappings between Segment and Amplitude APIs
+## Mappings between Segment and Amplitude SDK methods
 
-Segment and Amplitude use slightly different terms to describe the same concepts. The following table shows the mapping between the two APIs:
+Segment and Amplitude use slightly different terms to describe the same concepts. The following table shows the mapping between the two:
 
-| Segment's API | Amplitude's API | Description |
+| Segment SDK method | Amplitude SDK method | Description |
```

Comment:
Most of the time, API is referred to HTTP API. `logEvent` is a method of our legacy SDKs. We are comparing different method names of Segment and Amplitude SDKs here to give customers a more smooth and quicker switch. Moreover,  it can have a better search result. For example, in dev doc, if you search API, the toppest results are HTTP APIs.

Reply:
Got it, but we're talking about a public function here not a method (it can stand for private function). Maybe name it as `function` instead of `method`.

Reply:
Changed it to `functions`.